docs/src/overview/hardware.md

```diff
@@ -26,3 +26,16 @@ At the moment we support the following switch types:
     On our switches we run [Cumulus Linux](https://cumulusnetworks.com/products/cumulus-linux/). The metal-core writes network configuration specifically implemented for this operating system. Please also consider running Cumulus Linux on your switches if you do not want to run into any issues with networking.
 
     Of course, contributions for supporting other switch vendors and operating systems are highly appreciated.
+
+## Portable metal-stack Setup DIY
+
+A minimal physical hardware setup may contain at least the following components:
+
+|    | Vendor     | Series           | Model                | Function             |
+| :- | :--------- | :--------------- | :------------------- | :------------------- |
+| 2x | Edgecore   | AS5500 Series    | AS5512-54x (10G)     | leaf / exit switches |
+| 1x | Supermicro | Microcloud       | SYS-5039MA16-H12RFT  |                      |
+| 1x | Unifi      | Edgemax          | Edgerouter Pro       | front router for internet and out-of-band access to servers and switches |
```

Comment:
You need to make the first column one character wider and add a heading in order to properly render the table with Julia Markdown (don't ask me for the exact reason).

Here is a suggestion that works (including some typographical improvements):

```suggestion
| #   | Vendor      | Series           | Model                | Function                                                                 |
| :-- | :---------- | :--------------- | :------------------- | :----------------------------------------------------------------------- |
| 2x  | Edge-Core   | AS5500 Series    | AS5512-54x (10G)     | Leaf / Exit switches                                                     |
| 1x  | Supermicro  | Microcloud       | SYS-5039MA16-H12RFT  | Usable machines                                                          |
| 1x  | Unifi       | Edgemax          | Edgerouter Pro       | Front router for internet and out-of-band access to servers and switches |
```